<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WQ-20161107WOFJ" ignored="0" name="TestDebug" tests="1" failures="1" timestamp="06 一月 2017 10:40:00 GMT" time="13.136" errors="0">
  <testcase name="test1" time="1.041" classname="testng.Test1">
    <failure type="java.lang.AssertionError" message="检查点检查出错误">
      <![CDATA[java.lang.AssertionError: 检查点检查出错误
	at testng.CheckPoint.checkString(CheckPoint.java:56)
	at testng.Test1.test1(Test1.java:71)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- TestDebug -->
